src/main/java/org/sonar/plugins/findbugs/resource/DebugExtensionExtractor.java

```diff
@@ -70,7 +70,7 @@ public String getDebugSourceFromClass(InputStream classIn) throws IOException {
         protected String debug;
 
         public AbstractClassVisitor() {
-            super(Opcodes.ASM5);
+            super(Opcodes.ASM7);
```

Comment:
Wouldn't it be better to use `edu.umd.cs.findbugs.classfile.engine.asm.FindBugsASM.ASM_VERSION`, to prevent this from happening again in the future?